proto/serverless/instrumentation/tags/v1/error.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package serverless.instrumentation.tags.v1;
+
+option go_package = ".;protoc";
+
+message ErrorTags {
+    // The Error Name
+    string name = 1;
+    // The Error Message - Depending on runtime this is defined or not.
+    optional string message = 2;
+    // The Error stacktrace if applicable
+    optional string stacktrace = 3;
+
+    enum ErrorType {
+        ERROR_TYPE_UNSPECIFIED = 0;
```

Comment:
Let's put comments on what these mean in here. Unspecified looks new, and I haven't yet seen any docs on it.

Reply:
> Let's put comments on what these mean in here. Unspecified looks new, and I haven't yet seen any docs on it.

Yeah, my bad, I wrote it up but didn't push the commit out. Will get it updated. A note on `UNSPECIFIED` it is a requirement of enum types in protobuf. It functionally will never be seen in our data as our internal validation will reject that type, but it is needed so the type and client generation from the protobufs do not fail.